ui/components/app/assets/nfts/nft-details/nft-details.tsx

```diff
@@ -112,7 +111,10 @@ export default function NftDetails({ nft }: { nft: Nft }) {
   const isIpfsURL = nftSrcUrl?.startsWith('ipfs:');
   const isImageHosted =
     image?.startsWith('https:') || image?.startsWith('http:');
-  const nftImageURL = useGetAssetImageUrl(imageOriginal ?? image, ipfsGateway);
+  const nftImageURL = useGetAssetImageUrl(
+    (imageOriginal ?? image) as string | undefined,
```

Comment:
This type assersion is valid, since otherwise it would be treated as `string | undefined | null`

You could add an additional coalesce to make it undefined, but not sure if that would be weird.
```suggestion
    imageOriginal ?? image ?? undefined,
```

Reply:
https://github.com/MetaMask/metamask-extension/pull/29445/commits/e3ff91c2bd714efe6fef04f285871f81b84edb0d